pep-0644.rst

```diff
@@ -299,14 +311,32 @@ or extend compatibility with EOLed releases as we see fit.
 
 The new ABI stability and LTS policies of OpenSSL [9]_ should help, too.
 
+Keep support for OpenSSL 1.1.0
+------------------------------
+
+It was suggested to keep support for OpenSSL 1.1.0 for compatibility with
+Debian 9 (Stretch). The proposal was rejected since it would complicated code
+cleanup and testing. Stretch is already out of regular security support and
+close to end of long-term support. By the time of Python 3.10 final release,
+Debian Buster and Debian Bullseye will be available.
+
+Instead Python 3.10 will gain additional documentation and a new
```

Comment:
I suggest to elaborate slightly more on the configure options. But feel free to ignore this suggestion :)

Reply:
I have included ``--with-openssl-rpath=auto`` and a link to the BPO.